.gitignore

```diff
@@ -2,4 +2,8 @@
 mysql_config.json
 __pycache__
 .DS_Store
-.pre-commit-config.yaml
\ No newline at end of file
+.pre-commit-config.yaml
+dumps
+.pytest_cache
+mac_dats
+macresfork 2
```

Comment:
These files also need to go away.

Also, please fix your editor so it adds a new line at the end of files.